src/main/java/com/example/petbuddybackend/service/care/state/CareStateMachine.java

```diff
@@ -6,22 +6,29 @@
 import com.example.petbuddybackend.repository.care.CareRepository;
 import com.example.petbuddybackend.utils.exception.throweable.general.StateTransitionException;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.time.Duration;
+import java.time.LocalDate;
 import java.util.List;
 
 @Service
 @RequiredArgsConstructor
 public class CareStateMachine {
 
-    private static final String ACCEPTED_CARE_EDIT_MESSAGE = "Cannot edit care that has been accepted by caretaker";
+    private static final List<CareStatus> statusesThatCanBeCancelledBySystem =
```

Comment:
tak naprawde czym sie rozni accepted od ready to proceed, bo rozumiem ze jak oba statusy sa accepted to automatycznie zmienia sie na to drugie

Reply:
Odsyłam do wątku :) 

> zastanawiam sie czy jest nam do czegos potrzeby ten status ready to proceed. Chyba ze ulatwia on zweryfikowanie czy opieka jest zatwierdzona przez obie strony. Bo samo teraz zeby stwierdzic czy opieka jest zatwierdzona przez obu to mamy ready to proceed a tak trzebaby bylo sprawdzac oba statusy, wiec w sumie ma to sens bo nam jest latwiej xD

Reply:
Haha, lubię głośno myśleć xd